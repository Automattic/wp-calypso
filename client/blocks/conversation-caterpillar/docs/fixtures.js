/** @format */

/**
 * External dependencies
 */
import { keyBy, map } from 'lodash';

export const comments = [
	{
		ID: 1,
		content: '<p>Chocolate cake</p>',
		author: {
			ID: 1,
			name: 'Christophe',
			avatar_URL:
				'https://2.gravatar.com/avatar/5512fbf07ae3dd340fb6ed4924861c8e?s=96&d=identicon&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T14:50:33+00:00',
	},
	{
		ID: 2,
		content: '<p>Ice-cream cone</p>',
		author: {
			ID: 2,
			name: 'Boris',
			avatar_URL: 'https://2.gravatar.com/avatar/22d41e5b6ff197cd7900c0514d1bd305?d=mm&r=G',
			has_avatar: false, // Test that authors with has_avatar: false are omitted
		},
		date: '2016-04-18T14:50:33+00:00',
	},
	{
		ID: 3,
		content: '<p>Pickle</p>',
		author: {
			ID: 3,
			name: 'Matt Mullenweg',
			avatar_URL: 'https://1.gravatar.com/avatar/767fc9c115a1b989744c755db47feb60?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T14:59:22+00:00',
	},
	{
		ID: 4,
		content: '<p>Swiss cheese</p>',
		author: {
			ID: 1,
			name: 'Christophe',
			avatar_URL:
				'https://2.gravatar.com/avatar/5512fbf07ae3dd340fb6ed4924861c8e?s=96&d=identicon&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 5,
		content: '<p>I have no avatar, sad panda</p>',
		author: {
			ID: 1,
			name: 'Anon',
			avatar_URL: null,
			has_avatar: false,
		},
		date: '2016-04-18T15:23:00+00:00',
	},
	{
		ID: 6,
		content: '<p>Salami</p>',
		author: {
			ID: 6,
			name: 'Ben',
			avatar_URL: 'https://0.gravatar.com/avatar/3b7b2c457b9201d166b9a2b47cadc86d?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 7,
		content: '<p>Lollipop</p>',
		author: {
			ID: 7,
			name: 'Derek',
			avatar_URL: 'https://2.gravatar.com/avatar/2723d05b4e1176e977e1e6641a2ca98b?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 8,
		content: '<p>Cherry pie</p>',
		author: {
			ID: 8,
			name: 'Jake',
			avatar_URL: 'https://1.gravatar.com/avatar/7a6c0fad3d7e9d4038222cbadd095db8?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 9,
		content: '<p>Cupcake</p>',
		author: {
			ID: 9,
			name: 'Jan',
			avatar_URL:
				'https://secure.gravatar.com/avatar/3e44726f128c263db20117a659bfa003?d=mm&s=275&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 10,
		content: '<p>Watermelon</p>',
		author: {
			ID: 10,
			name: 'Aaron',
			avatar_URL: 'https://2.gravatar.com/avatar/20600ea4d39b6facdd79875d24d2dbab?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 11,
		content: '<p>Bagel</p>',
		author: {
			ID: 11,
			name: 'Greg',
			avatar_URL: 'https://0.gravatar.com/avatar/915ee4efbd21f12b55b6362cf4f7c42f?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 11,
		content: '<p>Apple</p>',
		author: {
			ID: 11,
			name: 'Greg',
			avatar_URL: 'https://1.gravatar.com/avatar/765b5dca3190e27b9c88d6143c5527d5?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 12,
		content: '<p>Pancake</p>',
		author: {
			ID: 12,
			name: 'Martin',
			avatar_URL: 'https://0.gravatar.com/avatar/33b3e384f5a150015a5a981402983ca3?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
	{
		ID: 13,
		content: '<p>Gelato</p>',
		author: {
			ID: 13,
			name: 'Yanir',
			avatar_URL: 'https://1.gravatar.com/avatar/dda019c47a6183120608a6aeac2db6c5?d=mm&r=G',
			has_avatar: true,
		},
		date: '2016-04-18T15:22:00+00:00',
	},
];

export const commentsTree = {
	...keyBy( comments, 'ID' ),
	children: map( comments, 'ID' ),
};
