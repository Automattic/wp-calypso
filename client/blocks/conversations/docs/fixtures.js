export const commentsTree = {
	1: {
		data: {
			ID: 1,
			content: '<p>Excellent!</p>',
			author: {
				name: 'Christophe',
				avatar_URL:
					'https://2.gravatar.com/avatar/5512fbf07ae3dd340fb6ed4924861c8e?s=96&d=identicon&r=G',
			},
			date: '2016-04-18T14:50:33+00:00',
		},
	},
	2: {
		data: {
			ID: 2,
			content: '<p>Tremendous!</p>',
			author: {
				name: 'Boris',
				avatar_URL: 'https://2.gravatar.com/avatar/22d41e5b6ff197cd7900c0514d1bd305?d=mm&r=G',
			},
			date: '2016-04-18T14:50:33+00:00',
		},
	},
	3: {
		data: {
			ID: 3,
			content: '<p>Splendid!</p>',
			author: {
				name: 'Matt',
				avatar_URL: 'https://1.gravatar.com/avatar/767fc9c115a1b989744c755db47feb60?d=mm&r=G',
			},
			date: '2016-04-18T14:59:22+00:00',
		},
	},
};
