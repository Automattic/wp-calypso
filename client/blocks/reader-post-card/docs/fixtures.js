/** @format */

/**
 * Internal dependencies
 */

import DisplayTypes from 'client/state/reader/posts/display-types';

export const posts = [
	{
		ID: 1,
		title: 'A Post Title',
		content: 'Look, I have content!',
		site_ID: 1,
		global_ID: 1,
		site_URL: 'http://example.com',
		URL: 'http://example.com/post1',
		feed_ID: 1,
		feed_item_ID: 1,
		author: {
			name: 'Sue Smith',
			email: 'sue@example.com',
		},
		discussion: {
			comment_count: 99,
		},
		canonical_media: {
			src: 'https://discover.files.wordpress.com/2017/02/cpgreber.jpg',
			width: 300,
			height: 200,
			mediaType: 'image',
		},
		date: '1976-09-15T10:12:00Z',

		site: 'cats.wordpress.com',

		better_excerpt:
			'Scamper destroy couch as revenge. Eat the cat food. Refuse to leave cardboard box meowzer! So get video posted to internet for chasing red dot in the house and running around all day...',
	},
	{
		ID: 2,
		title: 'A Post Title for Video Embed',
		content: 'Look, I have content!',
		site_ID: 2,
		global_ID: 2,
		site_URL: 'http://example.com',
		URL: 'http://example.com/post2',
		feed_ID: 2,
		feed_item_ID: 2,
		author: {
			name: 'Okapi Smith',
			email: 'okapi@example.com',
		},
		discussion: {
			comment_count: 42,
		},
		site: 'cats.wordpress.com',
		canonical_media: {
			aspectRatio: 1.641025641025641,
			autoplayIframe:
				'<iframe data-wpcom-embed-url="https://www.youtube.com/watch?v=ptkYu1fdRIM" class="youtube-player" type="text/html" width="640" height="390" src="https://www.youtube.com/embed/ptkYu1fdRIM?version=3&amp;rel=1&amp;fs=1&amp;autohide=2&amp;showsearch=0&amp;showinfo=1&amp;iv_load_policy=1&amp;wmode=transparent&amp;autoplay=1" allowfullscreen="true" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>',
			embedUrl: 'https://www.youtube.com/watch?v=ptkYu1fdRIM',
			height: 390,
			iframe:
				'<iframe data-wpcom-embed-url="https://www.youtube.com/watch?v=ptkYu1fdRIM" class="youtube-player" type="text/html" width="640" height="390" src="https://www.youtube.com/embed/ptkYu1fdRIM?version=3&amp;rel=1&amp;fs=1&amp;autohide=2&amp;showsearch=0&amp;showinfo=1&amp;iv_load_policy=1&amp;wmode=transparent" allowfullscreen="true" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>',
			src:
				'https://www.youtube.com/embed/ptkYu1fdRIM?version=3&rel=1&fs=1&autohide=2&showsearch=0&showinfo=1&iv_load_policy=1&wmode=transparent',
			thumbnailUrl: 'https://img.youtube.com/vi/ptkYu1fdRIM/mqdefault.jpg',
			type: 'youtube',
			mediaType: 'video',
			width: 640,
		},
		better_excerpt:
			'Scamper destroy couch as revenge. Eat the cat food. Refuse to leave cardboard box meowzer! So get video posted to internet for chasing red dot in the house and running around all day...',
	},
	{
		ID: 3,
		title: "I'm a photo only card!",
		display_type: DisplayTypes.PHOTO_ONLY,
		content: 'less than 140 characters of content to qualify',
		site_ID: 3,
		global_ID: 3,
		site_URL: 'http://example.com',
		URL: 'http://example.com/post3',
		feed_ID: 1,
		feed_item_ID: 1,
		author: {
			name: 'Sue Smith',
			email: 'sue@example.com',
		},
		discussion: {
			comment_count: 42,
		},
		canonical_media: {
			src: 'https://discover.files.wordpress.com/2015/03/empty-open-notebook-e1485546386849.jpg',
			mediaType: 'image',
			width: 300,
			height: 200,
		},
		date: '1976-09-15T10:12:00Z',
		site: 'cats.wordpress.com',
		short_excerpt: 'less than 140 characters of content',
	},
	{
		ID: 4,
		title: 'Gallery: Posters that will make you want to go vote',
		site_ID: 4,
		global_ID: 4,
		display_type: DisplayTypes.GALLERY,
		site_URL: 'http://example.com',
		URL: 'http://example.com/post4',
		feed_ID: 1,
		feed_item_ID: 1,
		author: {
			name: 'Sue Smith',
			email: 'sue@example.com',
		},
		canonical_media: {
			src:
				'https://tedideas.files.wordpress.com/2016/11/miltonglaser11x17.jpg?w=720&quality=80&strip=info',
			mediaType: 'image',
			width: 720,
			height: 988,
		},
		images: [
			{
				src:
					'https://tedideas.files.wordpress.com/2016/11/miltonglaser11x17.jpg?w=720&quality=80&strip=info',
				width: 720,
			},
			{
				src:
					'https://tedideas.files.wordpress.com/2016/11/gotv_posternataliawarren.jpg?w=720&quality=80&strip=info',
				width: 720,
			},
			{
				src:
					'https://tedideas.files.wordpress.com/2016/11/new-kind11x17_2.jpg?w=720&quality=80&strip=info',
				width: 720,
			},
			{
				src:
					'https://tedideas.files.wordpress.com/2016/11/jesusgarcia_aiga_gotv_yourvotematterseventothem.jpg?w=720&quality=80&strip=info',
				width: 720,
			},
		],
		discussion: {
			comment_count: 99,
		},
		date: '1976-09-15T10:12:00Z',
		site: 'cats.wordpress.com',
		better_excerpt:
			'Every four years, the Get Out the Vote campaign invites graphic designers to make posters that rally US voters to go to the polls. Here, 14 posters that rockâ€¦',
	},
];

export const feed = {
	blog_ID: 53424024,
	feed_ID: 41325786,
	name: 'Discover',
	URL: 'https://discover.wordpress.com/',
	feed_URL: 'http://discover.wordpress.com',
	subscribers_count: 10382083,
	is_following: false,
	image:
		'http://0.gravatar.com/blavatar/c9e4e04719c81ca4936a63ea2dce6ace?s=96&amp;d=http%3A%2F%2Fs2.wp.com%2Fi%2Fbuttonw-com.png',
	description:
		'A daily selection of the best content published on WordPress, collected for you by humans who love to read.',
};

export const site = {
	ID: 53424024,
	name: 'Discover',
	description:
		'A daily selection of the best content published on WordPress, collected for you by humans who love to read.',
	URL: 'https://discover.wordpress.com',
	jetpack: false,
	subscribers_count: 9665000,
	lang: false,
	icon: {
		img: 'https://secure.gravatar.com/blavatar/c9e4e04719c81ca4936a63ea2dce6ace',
		ico: 'https://secure.gravatar.com/blavatar/c9e4e04719c81ca4936a63ea2dce6ace',
	},
	logo: {
		id: 0,
		sizes: [],
		url: '',
	},
	visible: null,
	is_private: false,
	is_following: false,
};
