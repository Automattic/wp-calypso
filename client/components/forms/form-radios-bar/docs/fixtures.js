/** @format */
export const radios = [
	{
		label: 'First Radio',
		value: 'first',
	},
	{
		label: 'Second Radio',
		value: 'second',
	},
	{
		label: 'Third Radio',
		value: 'third',
	},
];

export const radiosWithThumbnails = [
	{
		label: 'First Radio',
		value: 'first',
		thumbnail: {
			cssClass: 'some-css-class',
		},
	},
	{
		label: 'Second Radio',
		value: 'second',
		thumbnail: {
			imageUrl: '/calypso/images/illustrations/illustration-empty-results.svg',
		},
	},
	{
		label: 'Third Radio',
		value: 'third',
		thumbnail: {
			cssColor: '#0087be',
		},
	},
];
