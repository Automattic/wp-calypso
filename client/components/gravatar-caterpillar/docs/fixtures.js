/** @format */

export const users = [
	{
		ID: 1,
		name: 'Christophe',
		email: 'christophe@example.com',
		avatar_URL:
			'https://2.gravatar.com/avatar/5512fbf07ae3dd340fb6ed4924861c8e?s=96&d=identicon&r=G',
		has_avatar: true,
	},
	{
		ID: 2,
		name: 'Boris',
		email: 'boris@example.com',
		avatar_URL: 'https://2.gravatar.com/avatar/22d41e5b6ff197cd7900c0514d1bd305?d=mm&r=G',
		has_avatar: false, // Test that authors with has_avatar: false are omitted
	},
	{
		ID: 3,
		name: 'Matt',
		email: 'matt@example.com',
		avatar_URL: 'https://1.gravatar.com/avatar/767fc9c115a1b989744c755db47feb60?d=mm&r=G',
		has_avatar: true,
	},
	{
		ID: 6,
		name: 'Ben',
		email: 'ben@example.com',
		avatar_URL: 'https://0.gravatar.com/avatar/3b7b2c457b9201d166b9a2b47cadc86d?d=mm&r=G',
		has_avatar: true,
	},
	{
		ID: 7,
		name: 'Derek',
		email: 'derek@example.com',
		avatar_URL: 'https://2.gravatar.com/avatar/2723d05b4e1176e977e1e6641a2ca98b?d=mm&r=G',
		has_avatar: true,
	},
	{
		ID: 8,
		name: 'Jake',
		email: 'jake@example.com',
		avatar_URL: 'https://1.gravatar.com/avatar/7a6c0fad3d7e9d4038222cbadd095db8?d=mm&r=G',
		has_avatar: true,
	},
	{
		ID: 9,
		name: 'Jan',
		email: 'jan@example.com',
		avatar_URL:
			'https://secure.gravatar.com/avatar/3e44726f128c263db20117a659bfa003?d=mm&s=275&r=G',
		has_avatar: true,
	},
	{
		ID: 10,
		name: 'Aaron',
		email: 'aaron@example.com',
		avatar_URL: 'https://2.gravatar.com/avatar/20600ea4d39b6facdd79875d24d2dbab?d=mm&r=G',
		has_avatar: true,
	},
	{
		ID: 11,
		name: 'Greg',
		email: 'greg@example.com',
		avatar_URL: 'https://0.gravatar.com/avatar/915ee4efbd21f12b55b6362cf4f7c42f?d=mm&r=G',
		has_avatar: true,
	},
	{
		ID: 12,
		name: 'Martin',
		email: 'martin@example.com',
		avatar_URL: 'https://0.gravatar.com/avatar/33b3e384f5a150015a5a981402983ca3?d=mm&r=G',
		has_avatar: true,
	},
	{
		ID: 13,
		name: 'Yanir',
		email: 'yanir@example.com',
		avatar_URL: 'https://1.gravatar.com/avatar/dda019c47a6183120608a6aeac2db6c5?d=mm&r=G',
		has_avatar: true,
	},
];
