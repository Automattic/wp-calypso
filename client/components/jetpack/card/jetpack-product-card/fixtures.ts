// TODO: replace this hardcoded data with real data when we have an endpoint for it or
// a more definitive place in our own codebase.
export const DEFAULT_UPGRADE_NUDGE_FEATURES = [
	{
		text: 'Real-Time Backup',
		description: 'Every edit gets saved immediatelly',
	},
	{
		text: 'Real-Time Scan',
		description: 'On-demand scanning',
	},
];
