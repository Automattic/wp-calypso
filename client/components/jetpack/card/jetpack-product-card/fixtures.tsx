/**
 * External dependencies
 */
import { translate } from 'i18n-calypso';
import React from 'react';

// TODO: replace this hardcoded data with real data when we have an endpoint for it or
// a more definitive place in our own codebase.
export const DEFAULT_UPGRADE_NUDGE_FEATURES = [
	{
		text: translate(
			'{{strong}}Backup {{em}}Real-time{{/em}}{{/strong}} Backs up your site as you edit',
			{
				components: {
					strong: <strong />,
					em: <em />,
				},
			}
		),
	},
	{
		text: translate( '{{strong}}Scan {{em}}Real-time{{/em}}{{/strong}} Always-on site protection', {
			components: {
				strong: <strong />,
				em: <em />,
			},
		} ),
	},
	{
		text: translate( 'All Jetpack security features for comprehensive protection' ),
	},
];
