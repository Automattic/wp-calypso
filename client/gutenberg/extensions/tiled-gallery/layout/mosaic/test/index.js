/**
 * External dependencies
 */
import React from 'react';
import { range } from 'lodash';
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import Mosaic from '..';
import * as imageSets from '../../test/fixtures/image-sets';

test( 'renders as expected', () => {
	Object.keys( imageSets ).forEach( k => {
		const images = imageSets[ k ];
		expect(
			shallow( <Mosaic images={ images } renderedImages={ range( images.length ) } /> )
		).toMatchSnapshot();
	} );
} );
