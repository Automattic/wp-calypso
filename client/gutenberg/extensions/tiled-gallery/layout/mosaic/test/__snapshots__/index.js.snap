// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<Gallery
  galleryRef={
    Object {
      "current": null,
    }
  }
>
  <Row
    key="0"
  >
    <Column
      key="0"
    >
      0
    </Column>
  </Row>
  <Row
    key="1"
  >
    <Column
      key="0"
    >
      1
    </Column>
  </Row>
  <Row
    key="2"
  >
    <Column
      key="0"
    >
      2
    </Column>
    <Column
      key="1"
    >
      3
    </Column>
    <Column
      key="2"
    >
      4
    </Column>
    <Column
      key="3"
    >
      5
    </Column>
  </Row>
  <Row
    key="3"
  >
    <Column
      key="0"
    >
      6
    </Column>
    <Column
      key="1"
    >
      7
    </Column>
  </Row>
  <Row
    key="4"
  >
    <Column
      key="0"
    >
      8
    </Column>
    <Column
      key="1"
    >
      9
      10
    </Column>
  </Row>
  <Row
    key="5"
  >
    <Column
      key="0"
    >
      11
      12
    </Column>
    <Column
      key="1"
    >
      13
    </Column>
  </Row>
</Gallery>
`;
